pkg/controller/qliksense/qliksense_controller.go

```diff
@@ -497,10 +500,23 @@ func (r *ReconcileQliksense) cronJobForGitOps(reqLogger logr.Logger, m *qlikv1.Q
 		},
 	}
 
+	updateCronJobForImageRegistry(m, cronJob)
 	controllerutil.SetControllerReference(m, cronJob, r.scheme)
 	return cronJob, nil
 }
 
+func updateCronJobForImageRegistry(m *qlikv1.Qliksense, cronJob *batch_v1beta1.CronJob) {
+	if imageRegistry := m.Spec.GetImageRegistry(); imageRegistry != "" {
+		podTemplateSpec := &cronJob.Spec.JobTemplate.Spec.Template.Spec
+		if currentImage := podTemplateSpec.Containers[0].Image; currentImage != "" {
+			imageSegments := strings.Split(currentImage, "/")
+			imageNameAndTag := imageSegments[len(imageSegments)-1]
+			podTemplateSpec.Containers[0].Image = fmt.Sprintf("%v/%v", imageRegistry, imageNameAndTag)
```

Comment:
what about this name `qlik-docker-oss.bintray.io/<NAMESPACE>/<PACKAGE_NAME>:<VERSION>` ?? assuming
`qlik-docker-oss.bintray.io/<NAMESPACE>` will be image registry value be provided

Reply:
@ffoysal: Today, our docker pull/push ignores paths in the URL and stores everything at the root of the repo.

Reply:
@ffoysal: to date, I have not run any tests with image registry specified containing paths, but it might work. Are you suggesting that we need to sanitize the path when we do substitution?

Reply:
got it, as discussed, user can provide image registry `registry:5000/foo-bar`